### PR TITLE
Fixed icons

### PR DESCRIPTION
**Pull request by Nodrex**

### CODE REVIEW

File: testomania/src/main/ic_launcher-playstore.png
Comment (shalva97) [inline testomania/src/main/ic_launcher-playstore.png:1]: this adds 80KB. Convert to bitmap

**Review by shalva97 — CHANGES_REQUESTED** [review]
its strange, why do you have pics in this PR

File: testomania/src/main/ic_launcher-playstore.png
Comment (Nodrex) [reply testomania/src/main/ic_launcher-playstore.png:1]: If you remember we have problem with icon on m device so I fixed it and that's why we have this PR

Comment (shalva97) [reply testomania/src/main/ic_launcher-playstore.png:1]: does it really help? this pic was meant to be uploaded to Play Store, its name also says that

Comment (Nodrex) [reply testomania/src/main/ic_launcher-playstore.png:1]: Yep on my device it helps, how about on yours? @shalva97

Comment (shalva97) [reply testomania/src/main/ic_launcher-playstore.png:1]: Can you convert them to webp?

Comment (Nodrex) [reply testomania/src/main/ic_launcher-playstore.png:1]: I am not sure that for the App icon, it is a good ide.
Are you sure that this will work?

Comment (shalva97) [reply testomania/src/main/ic_launcher-playstore.png:1]: yes, why would not it? is not it worse to have +80KB added to the app?

**Comment by Nodrex** [comment]
@shalva97 I am only able to convert the foreground launcher in to webp. others were skipped automatically when I tried conversation with android studio

**Comment by shalva97** [comment]
How did you convert it? Usually right click -> conver to webp works.

Any idea why Android Studio skips them?

**Comment by Nodrex** [comment]
> How did you convert it? Usually right click -> conver to webp works.
> 
> Any idea why Android Studio skips them?

Yep used right click -> convert to webp and I got message that it was skipped because gain of in size was 0 kb

Unsdfsdfsdftitled

**Review by shalva97 — APPROVED** [review]

**Comment by Nodrex** [comment]
> wait that image is not in resources folder, it wont be included in the app. Why you did not say it at the start :D

This is what I told fro the beginning :D This is an icon :D

**Pull request merged**